client/templates/discussions/comment/comment.js

```diff
@@ -43,9 +34,40 @@ Template.comment.events({
       }
     }).modal('show');
 
+  },
 
+  'click #edit-comment-icon': function(event, template) {
+    event.preventDefault();
+    let self = this;
+    Session.set('commentId', this._id);
 
+    Meteor.defer(function() {
```

Comment:
Yes it is, it's used in answers too, we did discuss this before I remember and it should be cleaned in next commit